package.json

```diff
@@ -3,10 +3,17 @@
   "description": "Client for Contentful's Content Delivery API",
   "version": "0.0.0-determined-by-semantic-release",
   "homepage": "https://www.contentful.com/developers/documentation/content-delivery-api/",
-  "main": "./dist/contentful.node.js",
-  "browser": "./dist/contentful.browser.js",
-  "modules": "./dist/contentful.browser.js",
+  "source": "./lib/index.ts",
+  "main": "./dist/contentful.cjs.js",
+  "module": "./dist/contentful.esm.mjs",
+  "exports": {
+    "import": "./dist/contentful.esm.mjs", 
+    "require": "./dist/contentful.cjs.js"
+  },
   "types": "./dist/types/index.d.ts",
+  "targets": {
```

Comment:
Is this the relevant part for Parcel to NOT compile the type definitions?

Reply:
It seems like both `targets` as well as `types` isn't relevant anymore as we don't create types with the help of parcel but rely exclusively on tsc.

Reply:
but `types` is pointing the consuming package to the types definitions, and therefore it's an essential field :)

Reply:
Ah, thanks, will revert then.

Reply:
And yes, seems that in that case we also need the `targets` property again :)